src/app/components/practice-container/practice-container.component.sass

```diff
@@ -0,0 +1,11 @@
+.practice-container
+    justify-content: center
+    align-content: center
+    margin: 48px
+    max-height: 80vh
+
+.button-container
+    display: flex
+    max-height: 20vh
+    justify-content: center
+    align-content: center
```

Comment:
If you are intending to center the button vertically and horizontally within the button-container, do you mean to use `align-items: center`?

(https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Aligning_Items_in_a_Flex_Container)

Reply:
yep i mistaken align-content as align-items oops